src/Panel/tabbedPanel.scss

```diff
@@ -1,4 +1,6 @@
-@import '../utils/theme/variables.scss';
+$tabbed-panel-tab-color: #3f4e49 !default;       // $gunmetal
```

Comment:
is this true even in terms of `.scss` (or better, `sass-loader`) semantics? In other words, are we sure sass-loader is compiling each top-level `.scss` import alone or we must take care of namespacing ourselves?